docs/setup_wsl.md

```diff
@@ -113,43 +118,75 @@ Enter a new UNIX username:
 
 </div>
 
-## Install CLI tools
-From an Ubuntu terminal, use the `apt` package manager to install a few command line programs.  Linux users will run this same command.
+### Install CLI tools
+From an Ubuntu bash terminal, use the `apt` package manager to install a few command line programs.
 ```console
 $ sudo apt update
 $ sudo apt install g++ make rsync wget git ssh gdb python3 tree
 ```
 {: data-variant="no-line-numbers" }
 
-## Use CLI tools
-Now would be a great time to take a look at our [CLI Tutorial](cli.html).
+### Home Directory
+You'll generally want to store your coding projects in the Linux filesystem, which is separate from your Windows filesystem.
 
-## Pro-tips
+Open an Ubuntu bash terminal and run `cd ~`. This will take you to your Ubuntu home directory. Running `pwd` afterward confirms the location.
```

Comment:
Does WSL default to your default user's home directory?  I hope so!

Reply:
Unfortunately, it depends how you launch it. Launching "Ubuntu" from the start menu takes you to `~` as it should, but launching "bash" from the start menu takes you to `/mnt/c/WINDOWS/system32` (ew).

So, the `cd ~` is here as a failsafe.